robomaker-sample-app-ci/src/aws-robomaker-sample-application-ci.ts

```diff
@@ -179,8 +184,9 @@ async function build() {
 
 async function bundle() {
   try {
+    const bundleFilename = WORKSPACE_DIRECTORY.split(path.sep).pop()    
```

Comment:
Is there a `basename()` function instead of doing a `split()` and `pop()`?

Reply:
Good point, testing basename now.

Reply:
Update: using basename now.